core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2395,10 +2405,13 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
-  public static final PropertyKey WORKER_TIERED_STORE_LEVEL2_DIRS_MEDIATYPE =
+  public static final PropertyKey WORKER_TIERED_STORE_LEVEL2_DIRS_MEDIUMTYPE =
```

Comment:
If there are more tiers, will the default still work?

Reply:
no they will not work any more